software/contrib/master_clock.py

```diff
@@ -219,7 +206,7 @@ def showScreen(self):
                 if abs(newBpm - self.state.get('bpm')) <= 10:
                     self.k2Unlocked = True
                 # update config value if k2 is unlocked and bpm has changed
```

Comment:
One last change and we're good to go.
```suggestion
                # update config value if k2 is unlocked
```